ingestion_server/README.md

```diff
@@ -121,12 +121,13 @@ database_table_to_elasticsearch_model = {
 }
 ```
 
-## Deployment (last deployed version: 1.4.0)
+## Deployment
 
-This codebase is deployed as Docker image to Docker hub. The deployed image is then pulled in the production environment. See the [`./publish_release.sh`](publish_release.sh) script for deploying to Docker hub. The machine running the script must be logged into Docker as the Openverse account.
```

Comment:
Can we delete the `publish_release.sh` file as part of this PR too?

Reply:
I made a separate issue for it #441 and a contributor has expressed interest.

Reply:
Ah right, thanks :smile: